components/NewRealmWizard/components/steps/CommunityTokenDetailsForm.tsx

```diff
@@ -95,7 +92,11 @@ export default function CommunityTokenForm({
     setValue,
     handleSubmit,
     formState: { isValid },
-  } = useForm({
+  } = useForm<any>({
```

Comment:
This seems like an improvement -- feel free to add a commit or change proposal. I personally would not spend time refactoring the wizards because I think rewriting from scratch would be a better use of time.